src/frequenz/sdk/actor/_channel_registry.py

```diff
@@ -28,6 +28,29 @@ def __init__(self, *, name: str) -> None:
         self._name = name
         self._channels: dict[str, Broadcast[typing.Any]] = {}
 
+    def set_resend_latest(self, key: str, resend_latest: bool) -> None:
```

Comment:
I always think of default as the safe value to use when the user doesn't care.  Here, if we make it default,  it feels like changing the meaning of the function - instead of setting the given value, it would be a `set/reset` thing, then we'll probably need a `reset_*` to be consistent.  

I feel users would call this only with a clear intention, so it is fine to expect that they specify what it is.

Reply:
Fair points